hw/ip/otp_ctrl/data/BUILD

```diff
@@ -78,6 +78,18 @@ otp_json(
                 "CREATOR_SW_CFG_DEFAULT_BOOT_DATA_IN_PROD_EN": "0x739",
                 # Enable AST initialization.
                 "CREATOR_SW_CFG_AST_INIT_EN": "0x6",
+                # Entropy source health check default values. This needs to be
+                # populated when `CREATOR_SW_CFG_RNG_EN` is set to true.
+                "CREATOR_SW_RNG_REPCNT_THRESHOLDS": "0xffffffff",
+                "CREATOR_SW_RNG_REPCNTS_THRESHOLDS": "0xffffffff",
+                "CREATOR_SW_RNG_ADAPTP_HI_THRESHOLDS": "0xffffffff",
+                "CREATOR_SW_RNG_ADAPTP_LO_THRESHOLDS": "0x0",
+                "CREATOR_SW_RNG_BUCKET_THRESHOLDS": "0xffffffff",
+                "CREATOR_SW_RNG_MARKOV_HI_THRESHOLDS": "0xffffffff",
+                "CREATOR_SW_RNG_MARKOV_LO_THRESHOLDS": "0x0",
+                "CREATOR_SW_RNG_EXTHT_HI_THRESHOLDS": "0xffffffff",
+                "CREATOR_SW_RNG_EXTHT_LO_THRESHOLDS": "0x0",
+                "CREATOR_SW_RNG_ALERT_THRESHOLD": "0xfffd0002",
```

Comment:
nit: I think all other items start with `CREATOR_SW_CFG`. Would you mind adding `_CFG` to these as well? If you think the names get too long we can shorten THRESHOLD as THOLD.

Reply:
Done.